auto_gptq/modeling/_base.py

```diff
@@ -760,6 +777,16 @@ def from_quantized(
             "_raise_exceptions_for_missing_entries": False,
             "_commit_hash": commit_hash,
         }
+        if not torch.cuda.is_available() or device == "cpu":
+            use_qbits = True
+
+        if use_qbits:
```

Comment:
i think we can add torch compatiblity check here, once detect not match throw a warning.